Resources/config/services.yml

```diff
@@ -12,8 +12,11 @@ services:
         class: Geocoder\ProviderAggregator
 
     Bazinga\GeocoderBundle\Plugin\FakeIpPlugin:
-      class: Bazinga\GeocoderBundle\Plugin\FakeIpPlugin
-      arguments: ['127.0.0.1', ~]
+        class: Bazinga\GeocoderBundle\Plugin\FakeIpPlugin
+        arguments: ['127.0.0.1', ~]
+
+    Bazinga\GeocoderBundle\Command\GeocodeCommand:
+        tags: [{ name: 'console.command', command: 'geocoder:geocode' }]
```

Comment:
Is `command: 'geocoder:geocode'` really needed when you use `$defaultName`?

Reply:
I took it from the doc: https://symfony.com/doc/current/console/commands_as_services.html#lazy-loading

I guess it's needed to be compatible with Symfony < 3.4

Reply:
Are you sure? 

The docs say "Use `$defaultName` OR set the command attribute on the console.command tag in your service definition"

Reply:
Yes you are right, sorry !